### PR TITLE
not accepting order amount if it isn't a number and equals 0

### PR DESCRIPTION
**Pull request by avsek477**
Checks for the value of order amount to be finite and not equals to 0 in ma crossover algo.

### CODE REVIEW

File: lib/ma_crossover/meta/validate_params.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ const validateParams = (args = {}) => {
   } = args
 
   if (!_includes(ORDER_TYPES, orderType)) return `Invalid order type: ${orderType}`
-  if (!_isFinite(amount)) return 'Invalid amount'
+  if (!_isFinite(amount) || amount === 0) return 'Invalid amount'
```
Comment (vitaliystoliarovcc) [inline lib/ma_crossover/meta/validate_params.js:44]: I think there should be a check: `amount > 0`

**Review by vitaliystoliarovcc — CHANGES_REQUESTED** [review]

File: lib/ma_crossover/meta/validate_params.js (hunk 1 above)
Comment (avsek477) [reply lib/ma_crossover/meta/validate_params.js:44]: so before we validate the params .. we are processing them first which negates the value if its a sell order .. so a check for amount equals 0 is fine here

Comment (vitaliystoliarovcc) [reply lib/ma_crossover/meta/validate_params.js:44]: I see. You are right

**Review by vitaliystoliarovcc — APPROVED** [review]
lgtm

**Review by robertkowalski — APPROVED** [review]
lgtm

**Pull request merged**